presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/Partition.java

```diff
@@ -33,6 +33,7 @@
 
 @Immutable
 public class Partition
+        extends org.apache.hadoop.hive.metastore.api.Partition
```

Comment:
Some of the fields in this class are already present in the superclass, wondering if we can remove those from this class and use from superclass?